### PR TITLE
DOC: add missing docstring for dim in plot_anat

### PR DESCRIPTION
**Pull request by mvdoc**

### CODE REVIEW

File: nipy/labs/viz_tools/activation_maps.py [hunk 1]
```diff
@@ -342,6 +342,8 @@ def plot_anat(anat=None, anat_affine=None, cut_coords=None, slicer='ortho',
             you whish to save figures with a black background, you
             will need to pass "facecolor='k', edgecolor='k'" to pylab's
             savefig.
+        dim: float, optional
+            If set, dim the anatomical image.
```
Comment (yarikoptic) [inline nipy/labs/viz_tools/activation_maps.py]: So is it opacity/alpha with 1 being in full, or amount of dimness ie 1-that?

Comment (mvdoc) [inline nipy/labs/viz_tools/activation_maps.py]: no, I believe it selects the maximum value for plotting to be sth from the mean above
```
vmax = vmean + (1+dim)*ptp
```

I didn't know how to phrase it precisely

**Comment by mvdoc** [comment]
ping @yarikoptic: better?

**Comment by yarikoptic** [comment]
Better than it was ;-)

**Pull request merged**